### PR TITLE
don't save newline when importing public keys via JWKS

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* Don't save newlines at the end of public keys when importing

**Release Note**

```release-note
Don't save newlines at the end of public keys when importing via JWKS
```

### CODE REVIEW

File: pkg/jwks/jwks.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func parseKeys(data []byte) ([]string, map[string]string, error) {
 		if err != nil {
 			return nil, nil, fmt.Errorf("marshal error: %w", err)
 		}
-		keys[i] = string(pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: encoded}))
+		keys[i] = strings.TrimSpace(string(pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: encoded})))
```
Comment (sethvargo) [inline pkg/jwks/jwks.go:139]: Should we copy over the project-level helper from the verification server instead? I'm fine with a copy-paste implementation.

**Review by sethvargo — APPROVED** [review]
/lgtm

/hold
for potentially more defense.

File: pkg/jwks/jwks.go (hunk 1 above)
Comment (mikehelmick) [reply pkg/jwks/jwks.go]: added... throughout the codebase

**Review by sethvargo — COMMENTED** [review]
/lgtm
/unhold

**Pull request merged**